packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -726,8 +779,24 @@ export class FeatureDevController {
     }
 
     private async stopResponse(message: any) {
-        const session = await this.sessionStorage.getSession(message.tabID)
-        session.state.tokenSource.cancel()
+        await telemetry.amazonq_stopCodeGeneration.run(async (span) => {
```

Comment:
I think I mentioned this in a different file but is there any reason we can't just use `telemetry.ui_click` here? since I believe this only being called when a user clicks on the stop generating button and tabID doesn't really give us anything valuable

Reply:
This change came from @kelvin-klchu, adding him here in the thread if makes sense or not.